app/models/miq_request_workflow.rb

```diff
@@ -143,8 +143,7 @@ def validate(values)
 
         # Check the disabled flag here so we reset the "error" value on each field
         next if dialog_disabled || fld[:display] == :hide
-
-        value = get_value(values[f])
+        value = fld[:data_type].to_s.match('array_') ? values[f] : get_value(values[f])
```

Comment:
Following rubocop here and suggest we change this to `=~` since we are not using the returned `MatchData` object.  It also removes the need for `to_s`.